Ark.Tools.AspNetCore.ApplicationInsights/Ark.Tools.AspNetCore.ApplicationInsights.csproj

```diff
@@ -24,10 +24,13 @@
   <ItemGroup>  
       <PackageReference Include="Microsoft.ApplicationInsights.AspNetCore" Version="2.21.0" />
       <PackageReference Include="Microsoft.ApplicationInsights.SnapshotCollector" Version="1.4.3" />
+      <PackageReference Include="Microsoft.AspNetCore.ApplicationInsights.HostingStartup" Version="2.2.0" />
```

Comment:
2.2.0 ... for an aspnetcore package which are generally version tied with AspNetCore version.

please check this is really needed and is latest for aspnetcore v6